test/components/collect-qr.spec.js

```diff
@@ -33,8 +34,9 @@ const qrData = (component) => {
 };
 
 describe('CollectQr', () => {
-  it('renders an image', () => {
-    const { childNodes } = mountComponent().element;
+  it('renders an image', async () => {
+    const { childNodes } = mountComponent().get('span').element;
+    await nextTick(); // wait for QR image to render in canvas
```

Comment:
With 2 children in the component (after adding the canvas tag, and no wrapper div), I was seeing four child nodes (though the first and last looked empty). I added this bit to get the span specifically and look at *its* children.